config/environments/development.rb

```diff
@@ -1,6 +1,10 @@
+require 'repository_container'
+require 'item_marketplace/active_record_repository/active_record_item_repository'
+
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
+  RepositoryContainer.add(:item, ActiveRecordRepository::ItemRepository.new)
```

Comment:
This setup should be in an initializer: http://guides.rubyonrails.org/configuring.html#locations-for-initialization-code